csharp.benchmark/FloatTimeSeriesBase.cs

```diff
@@ -7,33 +7,21 @@ public abstract class FloatTimeSeriesBase
     {
         protected static Column[] CreateFloatColumns()
         {
-            return new Column[]
-            {
-                new Column<DateTime>("DateTime", LogicalType.Timestamp(true, TimeUnit.Millis)),
-                new Column<int>("ObjectId"),
-                new Column<float>("Value")
-            };
+            return new Column[] {new Column<DateTime>("DateTime", LogicalType.Timestamp(true, TimeUnit.Millis)), new Column<int>("ObjectId"), new Column<float>("Value")};
```

Comment:
The previous style (one item per line) was more readable. Not sure why ReSharper felt like reformatting it on one line 😒
@ljubon can you please try to find a setting to avoid this reformatting?

Reply:
Thanks for pointing this out! It's a great example to test these tools further.
I'll check with ReSharper how to keep current re-formating with addition for this rule (not sure how to name it 🤔 )
Also, I would agree with choosing `dotnet-format ` as the final tool too, but I wanted to challenge it with `cleanupcode` where we have options in our hand, but as they say sometimes _less is more_ :)

Reply:
with a bit of tuning, it seems that all examples below (and others too) are reverted and added to `ParquetSharp.DotSettings` 👍 @jgiannuzzi @GPSnoopy  let me know if you find anything similar 👀